src/main/java/dk/kb/storage/storage/DsStorage.java

```diff
@@ -44,43 +48,57 @@ public class DsStorage implements AutoCloseable {
     private static final String CTIME_COLUMN = "ctime";
     private static final String MTIME_COLUMN = "mtime";
     private static final String PARENT_ID_COLUMN = "parentid";
-    private static final String KALTURA_REFERENCE_ID_COLUMN = "kalturareferenceid";
-    private static final String KALTURA_INTERNAL_ID_COLUMN = "kalturainternalid";
-    
-    
+    private static final String REFERENCE_ID_COLUMN = "referenceid";
+    private static final String KALTURA_ID_COLUMN = "kalturaid";
+    private static final String MAPPING_REFERENCE_ID_COLUMN = "referenceid";
+    private static final String MAPPING_KALTURA_ID_COLUMN = "kalturaid";
 
     private static String clearTableRecordsStatement = "DELETE FROM " + RECORDS_TABLE;
-
+    private static String clearTableMappingsStatement = "DELETE FROM " + MAPPING_TABLE;
+    
 
     private static String createRecordStatement = "INSERT INTO " + RECORDS_TABLE +
-            " (" + ID_COLUMN + ", " + ORIGIN_COLUMN + ", " +ORGID_COLUMN + ","+ RECORDTYPE_COLUMN +"," + IDERROR_COLUMN +","+ DELETED_COLUMN + ", " + CTIME_COLUMN + ", " + MTIME_COLUMN + ", " + DATA_COLUMN + ", " + PARENT_ID_COLUMN +  " , " + KALTURA_REFERENCE_ID_COLUMN +" , "+KALTURA_INTERNAL_ID_COLUMN+")"+
+            " (" + ID_COLUMN + ", " + ORIGIN_COLUMN + ", " +ORGID_COLUMN + ","+ RECORDTYPE_COLUMN +"," + IDERROR_COLUMN +","+ DELETED_COLUMN + ", " + CTIME_COLUMN + ", " + MTIME_COLUMN + ", " + DATA_COLUMN + ", " + PARENT_ID_COLUMN +  " , " + REFERENCE_ID_COLUMN +" , "+KALTURA_ID_COLUMN+")"+
             " VALUES (?,?,?,?,?,?,?,?,?,?,?,?)";
 
-    private static String updateRecordStatement = "UPDATE " + RECORDS_TABLE + " SET  "+			 
+
+    private static String createMapping = "INSERT INTO " + MAPPING_TABLE +
+            " (" + MAPPING_REFERENCE_ID_COLUMN + ", " + MAPPING_KALTURA_ID_COLUMN +")"+
+            " VALUES (?,?)";
+
+    private static String mappingByIdStatement = "SELECT * FROM " + MAPPING_TABLE + " WHERE "+ MAPPING_REFERENCE_ID_COLUMN+" = ?";
+    
+    private static String updateMappingStatement = "UPDATE " + MAPPING_TABLE + " SET  "+                         
+            KALTURA_ID_COLUMN + " = ?  "+            
+            "WHERE "+
+            MAPPING_REFERENCE_ID_COLUMN + "= ?";
```

Comment:
I am not sure if spaces are important here, but please add a space before the = sign